audiomentations/augmentations/air_absorption.py

```diff
@@ -0,0 +1,172 @@
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+import numpy as np
+import librosa
+
+
+def next_power_of_2(x: int) -> int:
+    """
+    taken jhoyla's answer here:
+    https://stackoverflow.com/questions/14267555/find-the-smallest-power-of-2-greater-than-or-equal-to-n-in-python
+    """
+    return 1 if x == 0 else 2 ** (x - 1).bit_length()
+
+
+class AirAbsorption(BaseWaveformTransform):
+    """
+    Applies a wind absorption transform parametrized by temperature, humidity, and distance
```

Comment:
Is it a wind or an air absorption transform ?

Reply:
Air, sorry.